> What am I supposed to do with MacOS compilation errors ?

fix reported error:
````
In file included from /Volumes/build/build/ci_checks/alice_ci_0/o2/sw/SOURCES/O2/6040/0/DataFormats/Detectors/CTP/src/Configuration.cxx:14:
/Volumes/build/build/ci_checks/alice_ci_0/o2/sw/SOURCES/O2/6040/0/DataFormats/Detectors/CTP/include/DataFormatsCTP/Configuration.h:53:3: error: unknown type name 'uint'
  uint HBaccepted; /// Number of HB frames in TF to be accepted
````

--- a/DataFormats/Detectors/CTP/include/DataFormatsCTP/Configuration.h
+++ b/DataFormats/Detectors/CTP/include/DataFormatsCTP/Configuration.h
@@ -50,13 +50,13 @@ struct CTPDescriptor {
 struct CTPDetector {
   CTPDetector() = default;
   o2::detectors::DetID::ID detID;
-  uint HBaccepted; /// Number of HB frames in TF to be accepted
+  uint32_t HBaccepted; /// Number of HB frames in TF to be accepted
   void printStream(std::ostream& strem) const;
 };
 struct CTPCluster {
   CTPCluster() = default;
   std::string name;
-  uint detectorsMask;
+  uint32_t detectorsMask;
   std::vector<std::string> detectorsNames;
   void printStream(std::ostream& strem) const;
   ClassDefNV(CTPCluster, 1)

--- a/DataFormats/Detectors/CTP/src/Configuration.cxx
+++ b/DataFormats/Detectors/CTP/src/Configuration.cxx
@@ -71,27 +71,27 @@ void CTPConfiguration::printStream(std::ostream& stream) const
 {
   stream << "Configuration:" << mName << std::endl;
   stream << "CTP BC  masks:" << std::endl;
-  for (const auto i : mBCMasks){
+  for (const auto i : mBCMasks) {
     i.printStream(stream);
   }
   stream << "CTP inputs:" << std::endl;
-  for (const auto i : mInputs){
+  for (const auto i : mInputs) {
     i.printStream(stream);
   }
   stream << "CTP descriptors:" << std::endl;
-  for (const auto i : mDescriptors){
+  for (const auto i : mDescriptors) {
     i.printStream(stream);
   }
   stream << "CTP detectors:" << std::endl;
-  for (const auto i : mDetectors){
+  for (const auto i : mDetectors) {
     i.printStream(stream);
   }
   stream << "CTP clusters:" << std::endl;
-  for (const auto i : mClusters){
+  for (const auto i : mClusters) {
     i.printStream(stream);
   }
   stream << "CTP classes:" << std::endl;
-  for (const auto i : mCTPClasses){
+  for (const auto i : mCTPClasses) {
     i.printStream(stream);
   }
 }

--- a/DataFormats/Detectors/CTP/src/Configuration.cxx
+++ b/DataFormats/Detectors/CTP/src/Configuration.cxx
@@ -71,21 +71,27 @@ void CTPConfiguration::printStream(std::ostream& stream) const
 {
   stream << "Configuration:" << mName << std::endl;
   stream << "CTP BC  masks:" << std::endl;
-  for (const auto i : mBCMasks)
+  for (const auto i : mBCMasks){
     i.printStream(stream);
+  }
   stream << "CTP inputs:" << std::endl;
-  for (const auto i : mInputs)
+  for (const auto i : mInputs){
     i.printStream(stream);
+  }
   stream << "CTP descriptors:" << std::endl;
-  for (const auto i : mDescriptors)
+  for (const auto i : mDescriptors){
     i.printStream(stream);
+  }
   stream << "CTP detectors:" << std::endl;
-  for (const auto i : mDetectors)
+  for (const auto i : mDetectors){
     i.printStream(stream);
+  }
   stream << "CTP clusters:" << std::endl;
-  for (const auto i : mClusters)
+  for (const auto i : mClusters){
     i.printStream(stream);
+  }
   stream << "CTP classes:" << std::endl;
-  for (const auto i : mCTPClasses)
+  for (const auto i : mCTPClasses){
     i.printStream(stream);
+  }
 }

